terracumber-cli

```diff
@@ -153,7 +155,7 @@ def run_terraform(args, tf_vars):
     """ Prepare the environment """
     terraform = terracumber.terraformer.Terraformer(args.gitfolder, args.tf,
                                                     args.sumaform_backend, tf_vars,
-                                                    args.logfile)
+                                                    args.logfile, args.terraformbin)
```

Comment:
If we change the name above, we need to change it here.

Reply:
done